src/backend/src/controllers/projects.controllers.ts

```diff
@@ -395,10 +395,16 @@ export default class ProjectsController {
 
   static async editLinkType(req: Request, res: Response, next: NextFunction) {
     try {
-      const { linkTypeId } = req.params;
-      const { iconName, required } = req.body;
+      const { linkTypePrevName } = req.params;
+      const { name: linkTypeNewName, iconName, required } = req.body;
```

Comment:
you shouldn't be able to edit the name since this uses the name as the id, if someone needs to change a linkType name, they should just be deleting and remaking it

Reply:
make sure to update all associated code in the backend